tests/testthat/test-tags.r

```diff
@@ -729,6 +729,31 @@ test_that("Latin1 and system encoding are converted to UTF-8", {
   )
 })
 
+test_that("paste8 in Chinese locale works", {
+  loc <- "Chinese"
+  testthat::skip_if_not(is_locale_available(loc), "Chinese locale not available")
```

Comment:
If `is_locale_available()` succeeds and returns TRUE, then the locale after this line will be `"Chinese"`. That means that after the `with_locale()` call below, the locale will still be `"Chinese"`.